cmd/genconversion/conversion.go

```diff
@@ -76,13 +82,19 @@ func main() {
 		}
 	}
 	generator.RepackImports(util.NewStringSet())
-	if err := generator.WriteImports(funcOut); err != nil {
+	if err := generator.WriteImports(data); err != nil {
 		glog.Fatalf("error while writing imports: %v", err)
 	}
-	if err := generator.WriteConversionFunctions(funcOut); err != nil {
+	if err := generator.WriteConversionFunctions(data); err != nil {
 		glog.Fatalf("Error while writing conversion functions: %v", err)
 	}
-	if err := generator.RegisterConversionFunctions(funcOut, fmt.Sprintf("%s.Scheme", apiShort)); err != nil {
+	if err := generator.RegisterConversionFunctions(data, fmt.Sprintf("%s.Scheme", apiShort)); err != nil {
 		glog.Fatalf("Error while writing conversion functions: %v", err)
 	}
+
+	b, err := imports.Process("", data.Bytes(), nil)
+	if err != nil {
+		glog.Fatalf("error while update imports: %v", err)
+	}
+	funcOut.Write(b)
```

Comment:
You should check that the write is successful. Omitting it is safe for data but this one can fail.